packages/react/src/Link/Link.tsx

```diff
@@ -28,15 +40,15 @@ export const Link = forwardRef(
         'amsterdam-link',
         {
           'amsterdam-link--inline': variant === 'inline',
-          'amsterdam-link--standalone-bold': variant === 'standaloneBold',
           'amsterdam-link--in-list': variant === 'inList',
+          'amsterdam-link--in-list__chevron': variant === 'inList' && !icon,
           'amsterdam-link--on-background-light': onBackground === 'light',
           'amsterdam-link--on-background-dark': onBackground === 'dark',
         },
         className,
       )}
     >
-      {variant === 'inList' && <Icon svg={ChevronRight} size="level-7" />}
+      {variant === 'inList' && <Icon svg={icon ? icon : ChevronRight} size="level-7" />}
```

Comment:
Perhaps inlist icon size could become a token? I thought you mentioned it could be one of 3 sizes.

Reply:
For `inList` links it can only be 16x16px (for the default chevron) or text level 7 (for all other icons). Icon sizing is still a bit messy though, we should discuss it with design